gateway/http.d/standalone.conf.liquid

```diff
@@ -0,0 +1,58 @@
+{% for server in standalone.server.listen %}
+server {
+    listen {{ server.port }} default_server
+    {%- if server.tls %} ssl {% endif -%}
+    {%- case server.protocol -%}
+    {%- when 'http2' %}{{- server.protocol -}}
+    {%- when 'spdy' %}{{- server.protocol -}}
+    {%- endcase -%}
+    {%- if server.proxy_protocol %} proxy_protocol {% endif -%}
+    ;
+
+    {% if server.tls.protocols -%}
+        ssl_protocols {{ server.tls.protocols | join: " " }};
+    {%- endif %}
+
+    {%- capture server_name -%}{{ server.name | default: 'default' }}{%- endcapture %}
+    set $port_name {{ server_name }};
+    server_name {{ server_name }};
+
+    set $ctx_ref -1;
```

Comment:
Why is this needed?

Reply:
It is used by `require('resty.ctx').stash()` to share the `ngx.ctx` between normal phases and post_action. apicast.conf does the same.